### PR TITLE
symcc: fix build

### PR DESCRIPTION
**Pull request by adrianherrera**
Use the latest Rust. Now SymCC builds.

### CODE REVIEW

**Comment by Alan32Liu** [comment]
Thanks, @adrianherrera!
I am also working on supporting `AFL++-SymCC` on `Ubuntu:20.04` (#1541 ), but there are failures like `undefined reference 'size_t'` when building it with benchmark programs.
Do you happen to know how to fix them? Thanks : )

**Comment by adrianherrera** [comment]
Hmm, I did not encounter that. But I was building with the current Docker images, which use Ubuntu 16.04

File: fuzzers/symcc_aflplusplus/builder.Dockerfile [hunk 1]
```diff
@@ -35,7 +35,14 @@ RUN cd /afl && unset CFLAGS && unset CXXFLAGS && \
     cp utils/aflpp_driver/libAFLDriver.a /
 
 # Install the packages we need.
-RUN apt-get install -y ninja-build flex bison python zlib1g-dev cargo 
+RUN apt-get install -y ninja-build flex bison python zlib1g-dev
```
Comment (jonathanmetzman) [inline fuzzers/symcc_aflplusplus/builder.Dockerfile:38]: Do an apt-get update first or there can  be issues.

Comment (adrianherrera) [reply fuzzers/symcc_aflplusplus/builder.Dockerfile]: Done.

**Pull request closed without merging**